tests/queries/0_stateless/03372_get_subcolumn_null.sql

```diff
@@ -0,0 +1,6 @@
+drop table if exists test;
+create table test (x Nullable(UInt32)) engine=Memory;
+insert into test select number % 2 ? null : number from numbers(10);
+select getSubcolumn(x, 'null') from test;
```

Comment:
Not familiar here, so confused,  - what is actually 'null' subcolumn for column `Nullable(UInt32)`? It doesn't look like a column which has subcolumns i.e., not a tuple, for example.

Reply:
It's a null map subcolumn. `Nullable` column is stored as 2 columns - UInt8 null map column and column with nested type and default values on rows with nulls. And we allow reading this null map column as `column.null` subcolumn. It's used for functions to subcolumns optimization to replace `isNull(nullable_column)` to `nullable_column.null` subcolumn to improve performance of the query and reduce the amount of read data.